addons/website/static/src/js/editor/widget_link.js

```diff
@@ -15,12 +15,17 @@ weWidgets.LinkDialog.include({
     custom_events: _.extend({}, weWidgets.LinkDialog.prototype.custom_events || {}, {
         website_url_chosen: '_onAutocompleteClose',
     }),
+    LINK_DEBOUNCE: 1000,
 
     /**
      * Allows the URL input to propose existing website pages.
      *
      * @override
      */
+    init: function () {
+        this._super.apply(this, arguments);
+        this._adaptPageAnchor = _.debounce(this._adaptPageAnchor, this.LINK_DEBOUNCE);
+    },
```

Comment:
Why adding the `init` method below the `start` documentation...? Can you please add it on top and add
```javascript
/**
 * @constructor
 */
```
like everywhere for the `init` method.